tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
```

Comment:
🤔 
Yeah, it's true and I also could have created all the other files by the same way actually (`file_put_content(json_encode(...))`), but I've done it like that.

I, personally, prefer commit two empty files to test a function than add some extra complexity to generate them.

But, anyway, does it really matter ??